codegen/templates/ApiClient.mustache

```diff
@@ -181,6 +181,11 @@ export class ApiClient {
             }
         }
 
+        // For ES modules, check for ReadStream using duck typing
```

Comment:
This is needed to make `'file': fs.createReadStream("test.txt"), ` work in our API call:

```

let attachmentsApiInstance = new Asana.AttachmentsApi();
let opts = { 
    'file': fs.createReadStream("test.txt"), 
    'parent': "123", 
    'name': "name_example_attach_node_js",
};
attachmentsApiInstance.createAttachmentForObject(opts).then((result) => {
    console.log('API called successfully. Returned data: ' + JSON.stringify(result.data, null, 2));
}, (error) => {
    console.error(error);
});

```